### PR TITLE
The connection object, created when opening a mqtt connection on the …

### PR DESCRIPTION
**Pull request by jontore**
…device

has a unsubscribe method

### CODE REVIEW

File: src/entities/Device.js [hunk 1]
```diff
@@ -149,6 +149,9 @@ default class Device {
             };
 
             mqtt.subscribe(newChannelCredentials.credentials.topic, connection.event);
+            connection.unsubscribe = (() => {
+                mqtt.unsubscribe(newChannelCredentials.credentials.topic);
```
Comment (SmbatYeranyan) [inline src/entities/Device.js]: How would this work if mqtt.unsubscribe doesn't exist in tools/mqtt.js as a method?

Comment (jontore) [inline src/entities/Device.js]: hehe, good question I assumed it was the PAHO client we got here.

Comment (SmbatYeranyan) [inline src/entities/Device.js]: Nppe :) its pointing to the mqtt class which has its own instance of paho within. The subscribe method is abstracted on top of the paho. So if you need to do unsubscribe, you should do something with the ` this._topics[topic].subscribers.push(eventCallback);` inside mqtt class

Comment (jontore) [inline src/entities/Device.js]: I added it :)

**Comment by SmbatYeranyan** [comment]
👍  awesome! good to go

**Pull request merged**